block-producer/src/block_builder/errors.rs

```diff
@@ -2,9 +2,8 @@ use miden_objects::accounts::AccountId;
 use miden_vm::{crypto::MerkleError, ExecutionError};
 use thiserror::Error;
 
-use crate::store::{ApplyBlockError, BlockInputsError};
-
 use super::prover::CREATED_NOTES_TREE_INSERTION_DEPTH;
+use crate::store::{ApplyBlockError, BlockInputsError};
 #[derive(Debug, Error, PartialEq, Eq)]
```

Comment:
I think there should be a blank line between these two lines.

Reply:
done